content/07.pathogenesis.md

```diff
@@ -334,33 +326,33 @@ Contact, droplet, and aerosol transmission are therefore all worth evaluating wh
+In an early study, viral nucleic acids were reported to remain at observable levels in the respiratory specimens of recovering hospitalized COVID-19 patients for a median of 20 days and with a maximum observed duration through 37 days, when data collection for the study ceased [@doi:10/ggnxb3].
 As more estimates of the duration of viral shedding are released, they are beginning to converge around approximately three weeks from first positive PCR test and/or onset of symptoms (which, if present, are usually identified within three days of the initial PCR test).
-For example, viral shedding was reported for up to 28 days following symptom onset by a second study [@doi:10.1038/s41586-020-2196-x] and for one to 24 days from first positive PCR test with a median of 12 days [@doi:10.1001/jama.2020.3204].
+For example, in later studies, viral shedding was reported for up to 28 days following symptom onset [@doi:10.1038/s41586-020-2196-x] and for up to 24 days from first positive PCR test, with a median of 12 days [@doi:10.1001/jama.2020.3204].
 On the other hand, almost 70% of patients were reported to still have symptoms at the time that viral shedding ceased, although all symptoms reduced in prevalence between onset and cessation of viral shedding (CVS) [@doi:10.1016/j.jcv.2020.104577].
-They also reported that the median time that elapsed between the onset of symptoms and cessation of viral RNA shedding (CVS) was 23 days and between first positive PCR test and CVS was 17 days [@doi:10.1016/j.jcv.2020.104577].
+The median time that elapsed between the onset of symptoms and cessation of viral RNA shedding (CVS) was 23 days and between first positive PCR test and CVS was 17 days [@doi:10.1016/j.jcv.2020.104577].
 The fact that this study reported symptom onset to predate the first positive PCR test by an average of three days, however, suggests that there may be some methodological differences between it and related studies.
 Furthermore, an analysis of residents of a nursing home with a known SARS-CoV-2 case measured similar viral load in residents who were asymptomatic regardless of whether they later developed symptoms, and the load in the asymptomatic residents was comparable to that of residents who displayed either typical of atypical symptoms [@doi:10.1056/NEJMoa2008457].
 Taken together, these results suggest that the presence or absence of symptoms are not reliable predictors of viral shedding or of SARS-CoV-2 status (e.g, [@doi:10.1001/jama.2020.6887].
-However, viral shedding is not necessarily indicative of contagiousness.
-The risk of spreading the infection was low after ten days from the onset of symptoms, as viral load in sputum was found to be unlikely to pose a significant risk based on their efforts to culture samples _in vitro_ [@doi:10.1038/s41586-020-2196-x].
+However, viral shedding is not necessarily indicative of contagiousness. 
```

Comment:
@SiminaB says:

> Seems like the answer to the above question is: No. However, we may want to mention this earlier, as it can be confusing. Note that we also use both the words “contagious” and “communicable.” – I think they’re synonyms.